build/rollup-config.js

```diff
@@ -0,0 +1,33 @@
+
+// Config file for running Rollup in "normal" mode (non-watch)
+
+// Needed to create files loadable by IE8
+import es3 from 'rollup-plugin-es3'
+import json from 'rollup-plugin-json'
+
+import gitRev from 'git-rev-sync'
+
+// TODO: There should be a way to skip the git branch+rev when doing a production build
```

Comment:
Yeah, but I'll wait for @mourner's input on the issue. This will affect the `build/publish.sh` script.